Kinvey/Kinvey/DataStore.swift

```diff
@@ -186,8 +186,17 @@ open class DataStore<T: Persistable> where T: NSObject {
      - returns: A `Request` instance which will allow cancel the request later
      */
     @discardableResult
-    open func find(_ id: String, readPolicy: ReadPolicy? = nil, completionHandler: @escaping ObjectCompletionHandler) -> Request {
-        return find(id, readPolicy: readPolicy) { (result: Result<T, Swift.Error>) in
+    open func find(
+        _ id: String,
+        readPolicy: ReadPolicy? = nil,
```

Comment:
I agree, the only catch is `readPolicy` and `writePolicy` are not available for all the requests that requestConfig would be present. We could either subclass `RequestConfiguration` or leave explicit in the docs which options are considered for each call. My opinion: subclass would be better since it make its clear what are the options available.